src/transformers/utils/fx.py

```diff
@@ -53,6 +54,7 @@
 
 
 logger = logging.get_logger(__name__)
+_IS_IN_DEBUG_MODE = os.environ.get("FX_DEBUG_MODE", False) in ["true", "True", "1"]
```

Comment:
Let's use `ENV_VARS_TRUE_VALUES` here from the `utils`:
```suggestion
_IS_IN_DEBUG_MODE = os.environ.get("FX_DEBUG_MODE", "").upper() in ENV_VARS_TRUE_VALUES
```